lib/config.js

```diff
@@ -340,6 +343,17 @@ config.start = function start() {
                          .pipe(errorLogger || logStream, {end: isPerWorker});
     worker.logFile = logStream;
   }
+
+  function setupExpresUsageForwarding(parentCtl) {
+    var debugExpress = require('./debug')('express-usage');
+    agent().on(
```

Comment:
I think the comment above must predate the conversation we just had in slack.

Anyhow, there should be unit tests, and once you pull this code into lib/express-stats.js, the debug will scoped on 'express-stats'. Not that you will need that for unit tests.